src/js/components/TableCell/TableCell.js

```diff
@@ -48,13 +48,39 @@ const TableCell = forwardRef(
     const tableContext = useContext(TableContext);
     const cellRef = useForwardedRef(ref);
     const containerRef = useRef();
+    const widthRef = useRef();
 
     useLayoutEffect(() => {
+      let resizeObserver;
+      const element = cellRef.current;
```

Comment:
Should this be 
```suggestion
      const element = cellRef?.current;
```

Reply:
cellRef is always defined. It's current may not be (although should be).